snuba/query/experiments.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional, Set
+
+from snuba import state
+from snuba.clickhouse.query_dsl.accessors import get_object_ids_in_query_ast
+from snuba.query.logical import Query as LogicalQuery
+
+
+def is_in_experiment(
+    query: LogicalQuery,
+    referrer: str,
```

Comment:
Let's make this nullable instead of relying on empty string to decide whether to filter by referrer or not. That would make it more straightforward.